Source/WebCore/html/HTMLAnchorElement.cpp

```diff
@@ -614,7 +615,7 @@ void HTMLAnchorElement::handleClick(Event& event)
             systemPreviewInfo.previewRect = child->boundsInRootViewSpace();
 
         if (auto* page = document().page())
-            page->handleSystemPreview(WTFMove(completedURL), WTFMove(systemPreviewInfo));
+            page->beginSystemPreview(completedURL, WTFMove(systemPreviewInfo), [keepBlobAlive = URLKeepingBlobAlive(completedURL)] { });
```

Comment:
I believe `document().topOrigin().data()` should be the second argument for constructing URLKeepingBlobAlive here